zebracrossing/campaigns/tests.py

```diff
@@ -20,6 +22,64 @@ def test_equals(self):
         self.assertEqual(time_slot, other)
 
 
+class SaveToTableTest(TestCase):
+    @classmethod
+    def setUp(self):
+        self.client = Client()
+        self.save_to_table_url = reverse("campaigns:save_to_table")
+        self.fp = open("README.md")
+        self.campaign = Campaign.objects.create(
+            client="Telkom", ad_agency="Telkom agency"
+        )
+        self.booking_sheet = BookingSheet(
+            ad_type="REC",
+            start_date=datetime.date(year=2022, month=6, day=20),
+            end_date=datetime.date(year=2022, month=7, day=31),
+            campaign=self.campaign,
+            booking_sheet=File(self.fp),
+            cost=23000,
+        )
+        self.booking_sheet.save()
+
+        self.time_slot = TimeSlot(time=datetime.time(hour=12, minute=53))
+        self.time_slot.save()
+
+        self.booked_day = BookedDay.objects.create(
+            date=datetime.date(2022, 6, 22),
+            timeslot=self.time_slot,
+            bookingsheet=self.booking_sheet,
+        )
+        self.booked_day.save()
+
+    def test_save_to_table(self):
+        ajax_data_expected = [
+            {"slot_time": "12:00", "date": "Tuesday (06/09)", "bookingsheet_id": "1"}
+        ]
+        response = self.client.post(
+            self.save_to_table_url,
+            json.loads(json.dumps(ajax_data_expected)),
```

Comment:
Why are you dumping json to a string and then loading it back to json from the string?